backends/qualcomm/utils/utils.py

```diff
@@ -223,7 +224,12 @@ def capture_program(
     core_ep.transform(ConvertBinaryOpsWithScalar())
     edge_ep = core_ep.to_edge(qnn_edge_config())
     _transform(edge_ep.exported_program)
-
+    # Since QDQ nodes are stripped, update graph signature again to validate program
```

Comment:
Not related to this diff - is it possible to run a mix of fp ops and quantized ops? Does it support well? The reason I'm asking is because we're removing all q/dq ops and the insert back to the i/o. It may limit us to do mixed dtypes.

Reply:
Currently mixed precision only supports for quantized ops since compiler spec for HTP precision (quantized or fp16) is on graph level granularity.
We'll have multi-graphs change in the near future, hopefully some mechanisms like weight sharing / fp mixed precision could be well-supported at that time.

Reply:
To do so, it would be great if the framework interface can provide runtime option (parallel with compiler spec) for backend to react: e.g. change performance config, select which graph in the context to be executed.

Reply:
Yeah we've been discussing how to pass the runtime option at the interface. Ideally passed it by the backend context. Question: do you need it to be `method::init` time or `method::execute` time?

Reply:
`method::execute` looks more flexible on QNN. Look forward to the change!